ghost/adapter-cache-redis/test/adapter-cache-redis.test.js

```diff
@@ -98,6 +99,40 @@ describe('Adapter Cache Redis', function () {
             }
         });
 
+        it('can wait for execution in the case of duplicate reads', async function () {
+            const KEY = 'wait-for-execution';
+            let cachedValue = null;
+            const redisCacheInstanceStub = {
+                get: function (key) {
+                    assert(key === KEY);
+                    return cachedValue;
+                },
+                set: function (key, value) {
+                    assert(key === KEY);
+                    cachedValue = value;
+                },
+                store: {
+                    getClient: sinon.stub().returns({
+                        on: sinon.stub()
+                    })
+                }
+            };
+            const cache = new RedisCache({
+                cache: redisCacheInstanceStub
+            });
+
+            const fetchData = sinon.stub().resolves('Da Value');
```

Comment:
I'm not sure how this test works reliably. Shouldn't we at least wait after both get calls before resoling the fetchData method?

Reply:
We don't need to do that because we're not `await`ing the calls to `get` - this test consistently fails without the changes and passes with them

Reply:
Okay, I'm still a bit skeptical still everything is still very synchronous and the executing already starts before awaiting the get calls, but we can just fix if it would ever start failing